### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by andreybolonin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,6 +30,10 @@ matrix:
           env: SYMFONY_VERSION='^2'
         - php: 7.1
           env: SYMFONY_VERSION='^3'
+        - php 7.1
```
Comment (dbu) [inline .travis.yml]: you need `- php: 7.1` (missing colon)
same with the next entry

Comment (andreybolonin) [reply .travis.yml]: ok, one moments, thx

Comment (andreybolonin) [reply .travis.yml]: tests passed

File: .travis.yml [hunk 2]
```diff
@@ -30,10 +30,13 @@ matrix:
...
         - php: 7.1
           env: SYMFONY_VERSION='^3'
+        - php: 7.1
+          env: SYMFONY_VERSION="4.0.*"
+        - php: 7.2
+          env: SYMFONY_VERSION="4.0.*"
 
 before_install:
     - if [ "$DEPENDENCIES" == "dev" ]; then composer config minimum-stability dev; fi;
-    - if [ "$SYMFONY_VERSION" != "" ]; then composer require symfony/lts:${SYMFONY_VERSION} --no-update; fi;
```
Comment (dbu) [inline .travis.yml]: this makes us ignore the SYMFONY_VERSION, we need to keep this line in

File: .travis.yml [hunk 3]
```diff
@@ -30,10 +30,13 @@ matrix:
           env: SYMFONY_VERSION='^2'
         - php: 7.1
           env: SYMFONY_VERSION='^3'
+        - php: 7.1
+          env: SYMFONY_VERSION="4.0.*"
+        - php: 7.2
+          env: SYMFONY_VERSION="4.0.*"
```
Comment (stof) [inline .travis.yml]: -1 for that. There is no reason to force testing the 4.0.x minor version explicitly. Currently, it is tested by normal jobs as being the latest stable version. And at the time these jobs are switching to Symfony 4.1, Symfony 4.0 will reach end of maintenance 2 months later, so not worth having us update this config every 6 months (and wait longer on Travis due to the quota on concurrent jobs per github organization).

The comment before these jobs says `Test LTS version`. 4.0 is not a LTS.

and btw, testing SF 4 on multiple PHP version version is even less relevant regarding our Travis quota

**Comment by stof** [comment]
I'm closing this PR. See my comment above about why it does not make sense.

**Pull request closed without merging**